### PR TITLE
Create package.json

### PR DESCRIPTION
**Pull request by NickWilde263**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "templated-hashmap",
+  "version": "2.0.1",
```
Comment (dleeds-cpi) [inline package.json:3]: I believe this should match the git tag name.
```suggestion
  "version": "v2.0.1",
```

**Review by dleeds-cpi — COMMENTED** [review]
Clib docs mentioned they preferred to name the file `clib.json`, so I'd recommend renaming it.

**Comment by NickWilde263** [comment]
renamed

**Review by dleeds-cpi — APPROVED** [review]

**Comment by dleeds-cpi** [comment]
Thanks. I have to approve with my other account. Standby.

**Review by DavidLeeds — APPROVED** [review]

**Pull request merged**